content/en/docs/languages/java/sampling.md

```diff
@@ -8,6 +8,57 @@ spans that are generated by a system. Which sampler to use depends on your
 needs. In general, decide which sampler to use at the start of a trace and allow
 the sampling decision to propagate to other services.
 
+## Default behavior
+
+By default, all spans are sampled, and thus, 100% of traces are sampled. If you
+do not need to manage data volume, don't bother setting a sampler.
```

Comment:
```suggestion
do not need to manage data volume, you don't need to set a sampler.
```

It would be nice to present typical situations representing "manage data volume".